packages/edit-site/src/components/layout/style.scss

```diff
@@ -118,7 +118,7 @@
 			// This ensure the radius work properly.
 			overflow: hidden;
 
-			@media (prefers-reduced-motion: no-preference) {
+			@media not (prefers-reduced-motion) {
```

Comment:
This code was reverted in #68464  because it caused errors in trunk, but it should work correctly in this PR.